[Elixir] Add analyzer comments for boutique-inventory (#2101)
exercism/website-copy: analyzer-comments/elixir/boutique-inventory/use_enum_sort_by.md

# Use `Enum.sort_by`

> One purpose of this exercise it to learn about sort_by. Use the `Enum.sort_by` function in the `sort_by_price` function to practice.

Reviewer: ```suggestion
Use the `Enum.sort_by` function in the `sort_by_price` function to practice. It's the best fitting `Enum` function for this task.
```
